Java/app/src/main/java/com/taboola/sdk4example/utils/Utils.java

```diff
@@ -0,0 +1,16 @@
+package com.taboola.sdk4example.utils;
+
+import android.content.res.AssetManager;
+import android.graphics.Typeface;
+
+import androidx.fragment.app.FragmentActivity;
+
+public class Utils {
+    private static final String FONTS_PATH = "fonts/";
+    private static final String FONT_SUFFIX = ".ttf";
+
+    public static Typeface loadFont(FragmentActivity activity, String fontName) {
```

Comment:
@DanielFrTB please align Kotlin and java project regarding typeface. in java you have created an util while in Kotlin no. in java you loaded typeface by calling Typeface.createFromAsset while in kotlin you used Typeface.Builder. just align you solution.